config/environments/test.rb

```diff
@@ -39,7 +39,7 @@
   config.action_mailer.default_options = {
     from: 'mail@example.com',
   }
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.default_url_options = { host: 'www.example.com', port: 3000 }
```

Comment:
What's the reason behind this change?

Reply:
That is only of tests, I thought that having to use `localhost:3000` inside the test was a bit weird.
But not strong opinion on that, possible I should keep that out, since this PR is already quite long.